db/migrate/20160614175042_translate_outdated_sequence_identifiers.rb

```diff
@@ -0,0 +1,25 @@
+class TranslateOutdatedSequenceIdentifiers < ActiveRecord::Migration
+  def up
```

Comment:
It might be a good idea to invoke curate:get_new_sequence_accessions as the first action in `up`.